.pre-commit-config.yaml

```diff
@@ -15,3 +15,9 @@ repos:
         stages: [commit-msg]
     repo: https://github.com/jorisroovers/gitlint
     rev: v0.18.0
+  - hooks:
+      - id: go-fmt-import
+      - id: go-vet
+      - id: go-lint
+    repo: https://github.com/Bahjat/pre-commit-golang
+    rev: v1.0.2
```

Comment:
1.0.3 is available, any reason not to use it?

Reply:
It is no longer being used, now we're using golangci-lint and using it's latest version.